docker-compose.yml

```diff
@@ -1,13 +1,13 @@
-version: "3.0"
+version: "3.4"
+
 services:
   web:
     build: .
     command: ./manage.py runserver 0.0.0.0:8000
     ports:
       - "8000:8000"
     environment:
-      - "SECRET_KEY=insecure - only for development"
-      - "MODE=DEVELOPMENT"
+      - "DJANGO_SETTINGS_MODULE=modernomad.settings.local"
```

Comment:
When using docker it'll always be in production? Or would you set this environment variable explicitly in production?

Reply:
[It is set explicitly as the default in `Dockerfile`.](https://github.com/embassynetwork/modernomad/blob/master/Dockerfile#L41) These settings in `docker-compose.yml` are the overrides just for the development environment.

Heroku doesn't look at `docker-compose.yml` at all, so it'll use the default from the image. We could also set it explicitly in the Heroku app config to be doubly sure it is correct, I suppose.